### ISSUE TITLE
v2.0.3 produces empty HTML page

### ISSUE THREAD

**Issue by strzibny** [bukalapak/snowboard#59]
I just downloaded and installed latest snowboard v2.0.3, but getting empty HTML output. To make sure the Blueprint is sane, I run it through drafter and snowboard lint. I also went and try a Blueprint that we used before in Apiary.io, so it seems there is a bug in the latest release?

1.7 works like a charm.

**Comment by subosito** [comment]
Hi @strzibny, thanks for your report.

Could you elaborate more on this and try using a blueprint from https://github.com/apiaryio/api-blueprint/tree/master/examples ?

**Comment by rtgoodwin** [comment]
I see the same, or something similar. With 2.0.3 and the example of "Advanced JSON Schema", most of the body is left out (see screenshot). I'm working with a very simple example of my own and wanting to build a custom template and again almost everything but the regular Markdown is left out. 
<img width="1132" alt="Advanced JSON Schema 2019-05-22 13-30-08" src="https://user-images.githubusercontent.com/1183842/58199348-e5ddcb00-7c95-11e9-8680-3dc9d6691b50.png">

**Comment by subosito** [comment]
Hi @strzibny and @rtgoodwin, yeah, there is some issue on API blueprint parsing during migration to nodejs. I've fixed that and just released a new version `v2.0.4` a moment ago.  

Could you please try that and see if it's working?

**Comment by subosito** [comment]
Going to close this issue for now, please reopen if needed.

**Comment by jjoker0110** [comment]
I'm getting the same issue, I just downloaded 3.0.0-rc.1 through npm and ran the "Hello World" Example in the notes and it is generating a blank page. 

If I open the developer tools, there is an error:

<img width="569" alt="Screen Shot 2019-07-15 at 10 33 19 AM" src="https://user-images.githubusercontent.com/1005541/61236077-fcc7f900-a6eb-11e9-9324-44b7fa7ab102.png">

My API.apib file is as follows:
```
# API
## GET /message
+ Response 200 (text/plain)

        Hello World!

```

And my command is:
`snowboard html -o output.html API.apib`

**Comment by subosito** [comment]
@jjoker0110 I’ve released `v3.0.0-rc.2` to fix the issue. Please let me know if the issue persists.
